crates/dekoder/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "dekoder"
+
+authors.workspace = true
+description.workspace = true
+readme.workspace = true
+categories.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[package.metadata.docs.rs]
+rustdoc-args = ["--html-in-header", "doc-header.html"]
+
+[dependencies]
+tar = "0.4.41"
+glob = "0.3.1"
+float-cmp = "0.9.0"
```

Comment:
I'm becoming more and more careful with dependencies, and `float-cmp` is only used for the `approx_eq!` macro. Maybe is worth to just replace it with a simple function, even defined here.

ULPS are fancy, but even a reasonable relative and absolute accuracies (in AND or OR) should be able to get the job done.
Even the library docs is saying:
> When I was a kid in the ’80s, the programming rule was “Never compare floating point numbers”. If you can follow that rule and still get the outcome you desire, then more power to you. However, ***if you really do need to compare them***, this crate provides a reasonable way to do so.
> https://docs.rs/float-cmp/latest/float_cmp/

(emphasis mine)

Reply:
done in c81bb08b